### DISCUSSION TITLE
Fold code at a certain level of indentation?

### DISCUSSION THREAD

**Discussion by nooblyf** [zed-industries/zed#8015] [Feature Requests]
Is there a way to fold code at a particular level, something like `cmd + k`, `cmd + 2` in vscode to fold all code at level 2 indendation?

**Comment by Moshyfawn** [comment]
There are keymaps for folding the immediate level of indentation.

```json
{
    "alt-cmd-[": "editor::Fold",
    "alt-cmd-]": "editor::UnfoldLines"
}
```

Multi-level indentation folding is still at the feature request phase. (ref: #5142)
